pid_controller/src/pid_controller.yaml

```
@@ -43,6 +43,11 @@ pid_controller:
     default_value: false,
     description: "Use external states from a topic instead from state interfaces."
   }
+  enable_feedforward: {
+    type: bool,
+    default_value: false,
+    description: "Enables feedforward gain on start."
+  }
```

I suggest renaming the parameter to `initial_state_feedforward_control` or similar, and set `read_only: true` for this parameter.

```suggestion
  initial_state_feedforward_control: {
    type: bool,
    default_value: false,
    read_only: false,
    description: "Enables feedforward control on startup."
  }
```